### ISSUE TITLE
[Modal/Sidepanel] Fields & Submit should be in HTML form

### ISSUE THREAD

**Issue by jeremie-lucca** [LuccaSA/lucca-front#2027]
Right now, it's impossible to have modal/sidepanel forms in a HTML form element, so teams tend to use a custom popup to have valid HTML.

**Comment by vvalentin-lucca** [comment]
One possible solution for the submit action would be to use the `form` attribute and pass it an identifier as a parameter. 
It's inexpensive, but it's a pain to use and not very robust…

I think a better solution would be to split up the component and allow `lu-modal-header` and `lu-modal-footer` to be nested within `lu-modal`.

(I also think `modal` is a bad name: not all dialog are modal).

An example in pseudo-code without footer:
```html
<lu-dialog>
	<lu-dialog-header />
	content
</lu-dialog>
```

An example in pseudo-code with a form:
```html
<lu-dialog>
	<lu-dialog-header />
	<form>
		content
		<lu-dialog-footer />
	</form>
</lu-dialog>
```

**Comment by jeremie-lucca** [comment]
New dialog service ✨
